lib/Controller/Campaign.php

```diff
@@ -625,7 +635,8 @@ public function assignLayout($campaignId)
         // Return
         $this->getState()->hydrate([
             'httpStatus' => 204,
-            'message' => sprintf(__('Assigned Layouts to %s'), $campaign->campaign)
+            'message' => sprintf(__('Assigned Layouts to %s'), $campaign->campaign),
+            'id' => $campaign->campaignId
```

Comment:
httpstatus 204 means no content, so I think we need to change this to 200 if we want to output the ID?